.github/workflows/PR-build.yml

```diff
@@ -24,116 +24,172 @@ env:
   PACKAGING_ROOT: build/packages 
 
 jobs:
+  changes:
+    runs-on: ubuntu-latest
+    outputs:
+      changed: ${{ steps.filter.outputs.changed }}
+    steps:
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          list-files: shell
+          filters: |
+            changed:
+              - '.github/workflows/PR-build.yml'
+              - 'cmd/**'
+              - 'config/**'
+              - 'deployment-template/**'
+              - 'e2etest/**'
+              - 'pkg/**'
+              - 'tools/**'
+              - 'Makefile'
+              - 'go.mod'
+              - 'go.sum'
```

Comment:
Nitpick but would it make sense to make these two `go.**`?

Reply:
Yeah doesn't hurt. I guess I technically don't even need to check go.mod since any go.mod change will result in a go.sum change as well.